components/button/demo/button-subtle.html

```diff
@@ -18,6 +18,7 @@ <h2>Subtle Button with Text Only</h2>
 			<d2l-demo-snippet>
 				<template>
 					<d2l-button-subtle id="normal" text="Subtle Button"></d2l-button-subtle>
+					<d2l-button-subtle slim id="normal" text="Slim Subtle Button"></d2l-button-subtle>
```

Comment:
Not sure why these had `id`s in the first place, but we don't want elements with the same `id` in the same shadow root. I think they can just be removed everywhere in this file.

Reply:
Good catch, I'm guessing they were copied from the visual-diff file originally.  Agreed that they aren't needed